### ISSUE TITLE
Open domain management responsibility 

### ISSUE THREAD

**Issue by crandmck** [expressjs/discussions#68]
IBM still owns and manages the https://expressjs.com domain.  The Express TC should at least have the ability to manage the domain, e.g. to configure HTTPS, etc.

cf. 
https://github.com/expressjs/expressjs.com/issues/955
https://github.com/expressjs/discussions/issues/60

**Comment by jasnell** [comment]
Sigh... This still hasn't been done? :( ... I've pinged Todd Moore about it again

**Comment by rubys** [comment]
I'm looking into this, will see if I can unblock it.

**Comment by rubys** [comment]
[How to move domains between Cloudflare accounts](https://support.cloudflare.com/hc/en-us/articles/204615358-How-to-move-domains-between-Cloudflare-accounts)

So...  who within The Express TC / foundation / whatever has access to the registrar for nameserver changes and access to a Cloudflare account?  I'm asking the same question within IBM, and hopefully we can connect the right people together.

**Comment by dougwilson** [comment]
I have access to _a_ registrar (but not the one that owns expressjs.com domain) if that helps. I can setup a Cloudflare account for the org to transfer into if that's the path forward :+1:

**Comment by rubys** [comment]
Re: _a_ registrar.  Point noted. :-).  Then again that means that the fire breathing dragons will be pointed your way if you don't share nicely with the Express TC.  Works for me.  :-)

Re: setup a Cloudflare account.  That seems to me to be a much easier approach than selling IBM on allowing non-IBMers access to an IBM Cloudflare account.

**Comment by jasnell** [comment]
@rubys ... When I had looked at this previously, it was quite difficult to identify who even had access to the IBM cloudflare account that managed the domain. Bonus points to you if you're able to quickly get a resolution on it :)

**Comment by rubys** [comment]
Oh, I've got the names and have gotten positive replies from both the people who can make the DNS change happen and the person who can make the Cloudflare account change happen.  :-)

The former will require executive approval (not a problem), and legal review, etc, and so the total elapsed time is a _minimum_ of a couple of weeks.  But as you are well aware, I can be persistent.

When we are ready, is @dougwilson the right contact from the foundation side?  My sense is that this needs to be held at foundation level, and not just "some guy who was active in 2018" (no disrespect intended to anybody, all of our interests change over time).

**Comment by dougwilson** [comment]
> will be pointed your way if you don't share nicely with the Express TC. 

> My sense is that this needs to be held at foundation level, and not just "some guy who was active in 2018" 

I'm not interested in being involved at this point :)

**Comment by jasnell** [comment]
@rubys ... not sure who has the keys to the Foundation's cloudflare account but @rvagg may be able to point you in the right direction. The foundation should be able to add it to their account but would need the @expressjs/express-tc to determine who needs to have administrative rights for it.

**Comment by mikeal** [comment]
The foundation owns nodejs.org https://www.whois.com/whois/nodejs.org

The build team, which is under the TSC, manages the DNS.

IBM should transfer the domain, if the express project doesn't have the
bandwidth the nodejs build team can probably help out. This should have
been transferred a year ago.

**Comment by rubys** [comment]
> IBM should transfer the domain

I will see to it.  Thanks!

**Comment by wesleytodd** [comment]
The express team has enough bandwidth to handle this, especially since it should not take more than an hour to hand it over and make sure everything works.  I am willing to do this it that is needed.  

As far as @dougwilson being the right person to handle this, he is the lead of the project, so I see no problem in handing it off to him personally.  I also do not see any problem having the nodejs foundation own it, but I just want to make it clear it is not a bandwidth limitation.

**Comment by mikeal** [comment]
The foundation should own the domain and have it pointed at DNS services
owned by the foundation. Pinging @kborchers to connect with the right LF
people to handle the transfer.

**Comment by mikeal** [comment]
To clarify, foundation should own the domain, express should manage the dns
servers. Didnt say that correctly in the last comment.

**Comment by rvagg** [comment]
👍 just make sure in the transfer that the DNS servers stay the same so you have smooth transfer. We can deal with Cloudflare account transfer afterward since it's just account-to-account. Maybe open an issue on nodejs/build when you need that step done. Rope me in if you need help elsewhere in the process.

**Comment by rubys** [comment]
I got word that the domain will be unlocked and I should have the authorization code within a matter of hours.  Who, specifically, should I work with?

**Comment by rubys** [comment]
We are ready to go from the IBM side to transfer the domain.

**Comment by crandmck** [comment]
@mikeal @kborchers ^^

**Comment by rubys** [comment]
Can this now be closed?

**Comment by crandmck** [comment]
Was the CloudFlare acct transferred?

**Comment by crandmck** [comment]
> Per the link above, The owner of the domain can add this domain to an existing (or new) CloudFlare account, and that will change the status of IBM's CloudFlare account for this domain to "Moved Away".

Since the foundation now owns the domain, it sounds like they need to do this, right?

> Is there any reason why this hasn't yet been done?

I guess that's a question for @mikeal or someone else in the Node/LF.

**Comment by dougwilson** [comment]
This is a long and old thread. Since it was opened, the domain has been moved to the foundation. There is still the questions about CloudFlare, etc., but if we _need_ to deal with that, let's open a new issue to at least make is easier to determine the current conversation :)
